From 8a92b009dc02e7c33cb916f8530236c636dd1d82 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Tue, 26 Feb 2019 17:05:58 +0100
Subject: [PATCH] cqm_debugfs: Fix compile warning

This function returns void, so we cannot return an error code here.
This function is only called by  cqm_deq_write() which already has this permission check. Juts remove the additional permission check here as it was already done.
---
 drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c b/drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c
index 5c63dff789d5..601d66ff1076 100644
--- a/drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c
+++ b/drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c
@@ -558,9 +558,6 @@ static inline void disp_deq_pon_reg(void *deq_base, u32 j)
 	unsigned long desc0 = DESC0_0_CPU_EGP_0;
 	int i;
 
-	if (!capable(CAP_SYS_ADMIN))
-		return -EPERM;
-
 	PRINTK("Name: CFG_PON_EGP_%02d\n", j);
 	PRINTK("Addr: 0x%8p\n",
 	       deq_base + CFG_CPU_EGP_0 + tmp);
